### PR TITLE
feat: Add Twitch stream to dashboard

### PR DESCRIPTION
**Pull request by kzalys**
Adds a Twitch stream for the greatunihack channel. Stolen from the `sh2020` branch.

### CODE REVIEW

File: src/views/pages/dashboard.ejs [hunk 1]
```diff
@@ -29,6 +32,24 @@
               </div>
             </div>
             <!-- USEFUL LINKS -->
+            <div class="col-xl-3 col-lg-4 col-sm-6">
+              <a href="https://twitch.com/greatunihack" target="_blank" rel="noopener noreferrer">
```
Comment (seanjparker) [inline src/views/pages/dashboard.ejs:36]: Here too

File: src/views/pages/dashboard.ejs [hunk 2]
```diff
@@ -16,6 +16,9 @@
 
       <div class="content">
         <div class="container-fluid">
+          <div id="twitch-player" class="text-center" style="display: none;">
+            <iframe src="https://player.twitch.tv/?channel=greatunihack&parent=hub.greatunihack.com" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe>
```
Comment (seanjparker) [inline src/views/pages/dashboard.ejs:20]: We should make an issue to not hardcode the channel name

**Review by seanjparker — APPROVED** [review]
LGTM % note in review

File: src/views/pages/dashboard.ejs (hunk 2 above)
Comment (kzalys) [reply src/views/pages/dashboard.ejs:20]: :+1: #154

**Pull request merged**